### PR TITLE
Added CSS Property: Padding

### PR DESCRIPTION
**Pull request by thePsguy**

### CODE REVIEW

**Comment by onebree** [comment]
+1, any update on this?

**Comment by thePsguy** [comment]
I'm active, and will fix any issues if there are any.

**Comment by onebree** [comment]
I meant any update as far as getting this merged :-)

**Comment by thePsguy** [comment]
Yep. (: I meant that for the contributers, in case they had left this request out for some reason.

File: css.html.markdown [hunk 1]
```diff
@@ -150,6 +150,24 @@ selector {
     min-width: 50mm;  /* millimeters */
     max-width: 5in;   /* inches */
     
+    /* Padding */
+            /*Set internal spacing in container element*/
```
Comment (vendethiel) [inline css.html.markdown]: can you please fix the indentation here&the merge conflict?

**Comment by vendethiel** [comment]
Sorry for the delay, we're trying to catch up :(.

**Comment by vendethiel** [comment]
Going to close this one for now :). Feel free to ping us here if you find some time working on it again! Thanks.

**Pull request closed without merging**